ibm_mq/datadog_checks/ibm_mq/ibm_mq.py

```diff
@@ -49,7 +55,18 @@ def __init__(self, *args, **kwargs):
         self.metadata_collector = MetadataCollector(self._config, self.log)
         self.stats_collector = StatsCollector(self._config, self.send_metrics_from_properties, self.log)
 
+        self.queue_manager_process_pattern = None
+
+        self.check_initializations.append(self.parse_config)
+
     def check(self, _):
+        if not self.check_queue_manager_process():
+            self.log.info('Process not found, skipping check run')
+            for sc_name in (self.SERVICE_CHECK, QueueMetricCollector.QUEUE_MANAGER_SERVICE_CHECK):
+                self.service_check(sc_name, self.UNKNOWN, self._config.tags, hostname=self._config.hostname)
```

Comment:
maybe add a message on this service check? Same message as the log line for example: `'Process not found, skipping check run'`